projects/hslayers/src/components/save-map/save-map.service.ts

```diff
@@ -427,7 +436,7 @@ export class HsSaveMapService {
       const definition = getDefinition(layer);
       if (definition && definition.url) {
         json.protocol = {
-          url: encodeURIComponent(definition.url),
+          url: `${definition.url}/geoserver/${getWorkspace(layer)}/wfs`,
```

Comment:
this change, i.e. commit 0c0a2e18366ba8bf79d12c75a56773611e964179 can go away from this PR as it is properly fixed by commit 494a21e94cf66e1a8cf7effea9c9b0c63e5e5e00 in #4330

Reply:
Dropped